cmd/cluster/support/common.go

```diff
@@ -32,14 +32,14 @@ func getLimitedSupportReasons(clusterId string) ([]*cmv1.LimitedSupportReason, e
 	//getting the cluster
 	cluster, err := ctlutil.GetCluster(connection, clusterId)
 	if err != nil {
-		return nil, errors.New(fmt.Sprintf("Can't retrieve cluster: %v\n", err))
+		return nil, -1, errors.New(fmt.Sprintf("Can't retrieve cluster: %v\n", err))
 	}
 
 	//getting the limited support reasons for the cluster
 	clusterLimitedSupportReasons, err := ctlutil.GetClusterLimitedSupportReasons(connection, cluster.ID())
 	if err != nil {
-		return nil, errors.New(fmt.Sprintf("Can't retrieve cluster limited support reasons: %v\n", err))
+		return nil, -1, errors.New(fmt.Sprintf("Can't retrieve cluster limited support reasons: %v\n", err))
 	}
 
-	return clusterLimitedSupportReasons, nil
+	return clusterLimitedSupportReasons, cluster.Status().LimitedSupportReasonCount(), nil
```

Comment:
fyi dupe of https://github.com/openshift/osdctl/pull/606#discussion_r1754311015 - I agree it's confusing, but `cluster.Status().LimitedSupportReasonCount()` doesn't count limited support reasons that have been overridden, so it can be different from `len(clusterLimitedSupportReasons)`, which doesn't take into account overrides.